src/buttons/buttonApply.command.js

```diff
@@ -132,36 +124,25 @@ export default class ButtonApplyCommand {
     });
   }
 
-  static getEmailForm() {
-    return mQuery('form[name=emailform]');
-  }
+  static setDefaultValues() {
+    const type = ButtonsService.getType();
```

Comment:
please use the existing: ContentService::getMode

Reply:
@volha-pivavarchyk

Reply:
I can't use ContentService. ButtonService.getFormType() returns either 'email' or 'form'

Reply:
you can check the type and then return email or form (or page)

Reply:
You are right